### PR TITLE
in_syslog: adding useful extra (trace, debug) logging

### PR DESCRIPTION
**Pull request by attila123**
Signed-off-by: Attila Vangel <vangel.attila@gmail.com>

### CODE REVIEW

**Comment by attila123** [comment]
These extra logs helped me to track down some problem of integrating with some syslog source. I think it is a good improvement in general, so I wish to contribute them.

File: plugins/in_syslog/syslog_prot.c [hunk 1]
```diff
@@ -78,6 +78,7 @@ int syslog_prot_process(struct syslog_conn *conn)
 
         /* Incomplete message */
         if (eof == end || (*eof != '\n' && *eof != '\0')) {
+            flb_debug("[syslog-prot] incomplete message! (Enable trace log to see the message.)");
```
Comment (edsiper) [inline plugins/in_syslog/syslog_prot.c]: would you please rename the message prefix to: [in_syslog] ..."

**Comment by edsiper** [comment]
ping

**Comment by edsiper** [comment]
timeout.

**Pull request closed without merging**